.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Report
+about: Report a bug to help improve SQLFluff
+labels: bug
+---
+
+## Expected Behaviour
+
+## Observed Behaviour
+
+## Steps to Reproduce
```

Comment:
should we add some structure here?

"If related to a parsing or linting issue, please include a minimal SQL example which reproduces the issues, along with the `sqlfluff parse ...` output, `sqlfluff lint ...` output and `sqlfluff fix ...` output when relevant"

Reply:
Good call 👍